server/utils/nunjucksSetup.js

```diff
@@ -71,4 +71,12 @@ module.exports = (app, path) => {
     }
     return value ? 'Yes' : 'No'
   })
+
+  njkEnv.addFilter('ifDefined', value => {
+    return value === undefined ? '' : ` - ${value}`
```

Comment:
this is too specific to have as a general macro and what it does is [surprising](https://wiki.c2.com/?PrincipleOfLeastAstonishment). 

All the other ways of formatting values appear inside ReportSummary, I think it would be better to be consistent and do this for this case.

Reply:
done